src/classes/particle.ts

```diff
@@ -332,32 +338,44 @@ export class Particle {
     //     img.src = url;
     // }
 
-    public grab() {
+    public grab(): void {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Declaration of public instance method not allowed after declaration of private instance method. Instead, this should come after constructors.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5011230)